tables/endpointsecurity/src/fileeventstableplugin.cpp

```diff
@@ -0,0 +1,171 @@
+#include "fileeventstableplugin.h"
+
+#include <chrono>
+#include <mutex>
+
+
```

Comment:
In case it's useful, all the initial code was formatted at each commit with clang-format using the following profile: https://github.com/zeek/zeek-agent/blob/master/.clang-format

This should automatically take care of most issues like stray lines and wrong indentation

Reply:
Thanks! I'll definitely use that for future commits.